src/fsharp/FSharp.Core/option.fsi

```diff
@@ -33,7 +43,11 @@ module Option =
     /// <param name="option">The input option.</param>
     ///
     /// <returns>The option if the option is Some, else the default value.</returns>
-    /// <remarks>Identical to the built-in <see cref="defaultArg"/> operator, except with the arguments swapped.</remarks>
+    ///
+    /// <remarks>Identical to the built-in <see cref="defaultArg"/> operator, except with the arguments swapped.
+    /// <code>
+    /// None |> Option.defaultValue 99 = 99
```

Comment:
I'm concerned the "= 99" gives something that isn't easily copy-and-paste.   Do you think we could use "// 99" or a more obvious  unicode arrow character here?  (the latter is also not copy-and-paste but might still be more obvious to users)

Reply:
If these I think I prefer `// evaluates to 99`

Reply:
Ack.

Reply:
If copy-paste is a goal, will value-restriction be a problem? There are a number of these examples that will run into value restriction if executed standalone, e.g. None |> Option.toList. Currently I'm choosing _NOT_ to add type annotations to them because I think it gets in the way of clarity, and because value restriction isn't always applicable. Should I change this?

Reply:
> If copy-paste is a goal, will value-restriction be a problem?

Yes, you're right, and you're making the right decisions here